content/en/docs/concepts/architecture/garbage-collection.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Garbage Collection
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+{{<glossary_definition term_id="garbage-collection" length="short">}} This
+allows the clean up of resources like the following:
+
+  * [Objects without owner references](#owners-dependents)
+  * [Unused containers and container images](#containers-images)
+  * [Failed pods](/docs/concepts/workloads/pods/pod-lifecycle/#pod-garbage-collection)
+  * [Dynamically provisioned PersistentVolumes with a StorageClass reclaim policy of Delete](/docs/concepts/storage/persistent-volumes/#delete)
+  * [Stale or expired CertificateSigningRequests (CSRs)](/reference/access-authn-authz/certificate-signing-requests/#request-signing-process)
+  * Nodes deleted in the following scenarios:
+    * On a cloud when the cluster uses a [cloud controller manager](/docs/concepts/architecture/cloud-controller/)
+    * On-premises when the cluster uses an addon similar to a cloud controller
+      manager
+  * [Node Lease objects](/docs/concepts/architecture/nodes/#heartbeats)
+  * [Completed Jobs](/docs/concepts/workloads/controllers/ttlafterfinished/)
+
+## Owners and dependents {#owners-dependents}
+
+Many objects in Kubernetes are linked with each other through [*owner references*](/docs/concepts/overview/working-with-objects/owners-dependents/). 
+Owner references tell the control plane which objects are dependent on others.
+You can use owner references to block garbage collection of dependent resources
+before their owner objects are deleted. In most cases, Kubernetes manages owner
+references automatically.
+
+## Cascading deletion {#cascading-deletion}
+
+Kubernetes checks for and garbage collects objects that no longer have owner
+references, like the pods left behind when you delete a ReplicaSet. When you
+delete an object, you can control whether Kubernetes deletes the object's
+dependents automatically, in a process called *cascading deletion*. There are
+two types of cascading deletion, as follows: 
+
+  * Foreground cascading deletion
+  * Background cascading deletion
+
+You can also control how and when garbage collection deletes resources that have
+owner references using Kubernetes {{<glossary_tooltip text="finalizers" term_id="finalizer">}}. 
+
+### Foreground cascading deletion
+
+In foreground cascading deletion, the owner object first enters a *deletion in
+progress* state. In this state, the following happens to the owner object:
+
+  * The object is still visible through the API
```

Comment:
```suggestion
* The object starts off as visible through the API
```

Reply:
Converting to an ordered list to see if that makes it clearer.